python_ta/transforms/type_inference_visitor.py

```diff
@@ -196,6 +196,12 @@ def _closest_frame(self, node, name):
 
     def visit_name(self, node):
         try:
+            node.type_constraints = TypeInfo((self.type_constraints.lookup_concrete(
+                self._closest_frame(node, node.name)).type_environment.lookup_in_env(node.name)))
+        except KeyError:
```

Comment:
I don't know what's going on here exactly - possibly a bad merge. This `except` clause catches the same error as the following one.